data_hub_api/docmaps/v2/docmap_typing.py

```diff
@@ -72,15 +72,15 @@
     }
 )
 
-DocmapPreprintInputWithPublishedTdmpath = TypedDict(
-    'DocmapPreprintInputWithPublishedTdmpath',
+DocmapPreprintInputWithPublishedMecapath = TypedDict(
```

Comment:
should maybe use a capital P for path? i.e. `DocmapPreprintInputWithPublishedMecaPath`